modules/nodes-pods-secrets-creating-sa.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-sa_{context}"]
+= Creating a service account token secret
+
+As an administrator, you can create a service account token secret, which allows for distribute a service account’s token to external applications that must authenticate to the API.
+
+[NOTE]
+====
+It is recommended to obtain tokens using the TokenRequest API instead of using service account token secrets. The tokens obtained from the TokenRequest API are more secure than the tokens stored in secrets, because they have a bounded lifetime and are not readable by other API clients.
```

Comment:
Hi Michael, the customer suggests to add a link here to redirect to "TokenRequest API" page https://docs.openshift.com/container-platform/4.10/authentication/bound-service-account-tokens.html because we suggest to use TokenRequest API so we'd better provide convenience to the user.